src/index.ts

```diff
@@ -138,6 +139,14 @@ bundle
   .action(listBundle)
   .option('-a, --apikey <apikey>', 'apikey to link to your account');
 
+bundle
+    .command('current [channel]')
```

Comment:
Since we get the bundle of a channel, I would maybe add this command to channel instead